src/main/webapp/features/screens/game/Game.controller.js

```diff
@@ -1,20 +1,27 @@
 'use strict';
 
 angular.module('catan')
-    .controller('GameController', ['$scope', '$stateParams', '$state', 'Remote', function($scope, $stateParams, $state, Remote) {
+    .controller('GameController', [
+            '$scope', '$stateParams', '$state', 'Game',
+            function($scope, $stateParams, $state, Game)
+    {
 
-        $scope.gameDetails = null;
+        var GAME_UPDATE_DELAY = 5000;
 
-        $scope.updateGameDetails = function() {
-            Remote.game.details({gameId: $stateParams.gameId})
-                .then(function(response) {
-                    $scope.gameDetails = response.data;
-                }, function(response) {
-                    alert('Getting Game Details Error: ' + ((response.data.errorCode) ? response.data.errorCode : 'unknown'));
-                    $state.go("start");
-                });
-        };
+        $scope.game = Game(+$stateParams.gameId);
 
-        $scope.updateGameDetails();
+        $scope.game.load()
```

Comment:
I don't like that we call load() method of model to update this model, maybe we should have separate service, that load and updates model?